bcl2fastq/index_summary.py

```diff
@@ -127,13 +185,21 @@ def update_known_index_bar(run_json):
     [dep.Input('pruned_unknown_hidden', 'children')]
 )
 def update_unknown_index_pie(run_json):
```

Comment:
Almost there. I would update this function name to reflect the fact that it deals with a bar graph rather than a pie chart.